Makefile

```diff
@@ -26,7 +35,7 @@ install:
 	cp runc /usr/local/bin/runc
 
 clean:
-	rm runc
+	rm runc || true
```

Comment:
i will update this tomorrow, you all can chill ;)

On Mon, Sep 7, 2015 at 8:24 PM, W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/runc/pull/244#discussion_r38888021>:
>
> > @@ -26,7 +35,7 @@ install:
> >  	cp runc /usr/local/bin/runc
> >
> >  clean:
> > -	rm runc
> > +	rm runc || true
>
> On Mon, Sep 07, 2015 at 07:56:45PM -0700, Lai Jiangshan wrote: > - rm runc
> > + rm runc || true I think the intention of the author is that " if the
> file runc is non-existed, it shouldn't fail".
> rm(1) has:
> … <#14faafbb7d161b07_>
> -f, --force ignore nonexistent files and arguments, never prompt so
> there's no need for anything fancier than ‘rm -f runc’.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runc/pull/244/files#r38888021>.
>